dashboard/src/shared/AxiosInstance.ts

```diff
@@ -51,3 +62,18 @@ export function createAxiosInterceptors(axios: AxiosInstance, store: Store<IStor
     },
   );
 }
+// createAxiosInterceptors will configure a set of interceptors to a provided axios instance,
+// relying also on an external redux store for action dispatching
+export function createAxiosInterceptors(axiosInstance: AxiosInstance, store: Store<IStoreState>) {
```

Comment:
IMO this and `createAxiosInterceptorsWithAuth` are more difficult to read than simply exporting and calling `addErrorHandling` and `addAuthHeaders` from index https://github.com/kubeapps/kubeapps/pull/995/files#diff-ebf3c79d519804e9e573437d0db1b265R12